extensions/wikia/GlobalFooter/styles/GlobalFooterOasis.scss

```diff
@@ -1,113 +1,98 @@
-@import 'skins/shared/mixins/clearfix';
 @import 'skins/shared/color';
+@import 'skins/shared/mixins/box-sizing';
+@import 'skins/oasis/css/core/responsive-variables';
 
 $branding-color: #000;
+$global-footer-color-border: #c7d1d8;
+$global-footer-vertical-spacing: 30px;
 
 .global-footer {
-	box-sizing: content-box;
-	clear: left;
-	font-size: 11px;
-	padding-bottom: 10px;
-
-	nav {
-		@include clearfix;
-		margin-top: 15px;
-		position: relative;
+	@include box-sizing(border-box);
+
+	*, *::after {
```

Comment:
@hakubo We're doing it for Venus skin only globally. It's neat idea to do this by default to Oasis too, tho.